Sources/Benchmarks/ConcurrencyPlatform.swift

```diff
@@ -0,0 +1,211 @@
+// Copyright 2020 Penguin Authors
```

Comment:
Consider adding a pointer to the GitHub issue to provide context for why this (and related) files are dupe'd here.